google/cloud/bigtable/tools/run_emulator_utils.sh

```diff
@@ -82,10 +82,12 @@ function start_emulators() {
   trap kill_emulators EXIT
 
   local -r CBT_EMULATOR_CMD="/usr/local/google-cloud-sdk/platform/bigtable-emulator/cbtemulator"
+  rm -f emulator.log
```

Comment:
I don't think this can make any difference.  Whether `emulator.log` exists or not here, the next command will `O_CREAT|O_TRUNC` it before `CBT_EMULATOR_CMD` is executed.

Reply:
Fixed.

Reply:
I think the key insight in this PR is that the invocation of the emulator is done in the background, and the `emulator.log` file is opened (and truncated) also in that background process. So that creates a race condition with the latter function that reads the `emulator.log` file looking for a port number, since the backgrounded process may not have truncated the log yet. The fix proposed here is to remove the log file before running the emulator in the background.

This seems like a reasonable fix to me, and it seems like it applies here just as much as in the other emulator startup scripts in this PR.

But I apologize if I've misunderstood the comment or situation here.

Reply:
I apologize ... I missed the location of the fork.  I agree that the (re)creation of the log file happens in the child process the way things stand, and therefore that removing them first indeed has an effect.

Reply:
In the specific case of the **bigtable** emulator I was over enthusiastic and @devbww is right, there is no reading back of the log file, so removing it is not useful.  In the other emulators it seems to matter.